keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -984,26 +1008,31 @@ First, there will be no effect on any workload that doesn't use a new field. Any
 combination of feature gate enabled/disabled or version skew will work as usual
 for that workload.
 
-So when the new functionality wasn't yet used, downgrade will not be affected.
+When the new functionality wasn't yet used, downgrade will not be affected.
 
 Due to the new field added to `initContainers` to turn them into sidecars,
 downgrading to the version without this feature will make all Pods using this
-flag unscheduleable. New Pods will be rejected by the control plane and
-all kubelets. Pods that has already been created will not be rejected by control
-plane, but once reaching the kubelet, that has this feature disabled or which
-is old, kubelet will reject the Pod on unmarshalling.
-
-**Note**, we tested kubelet behavior. For the control plane we may need
-to implement a new logic to reject such Pods when feature gate got turned off.
+flag unscheduleable. New Pods will be rejected by the control plane.
```

Comment:
Are they really rejected? Or are we actually silently dropping them?
(e.g. https://github.com/kubernetes/kubernetes/blob/master/pkg/api/pod/util.go#L476-L477 )

Reply:
This is what I see when I run kubectl:

> Error from server (BadRequest): error when creating "STDIN": Pod in version "v1" cannot be handled as a Pod: strict decoding error: unknown field "spec.initContainers[0].restartPolicy"

So I assume it is rejected.

Reply:
Can you please check that not via running kubectl, but rather by sending API call?

Reply:
You are correct. The API server just stripped the unknown field.

Reply:
Updated text.